mica/report/report.py

```diff
@@ -843,7 +843,7 @@ def update():
 
     recent_obs = np.unique(fetch_states(start=DateTime(-7), vals=['obsid'])['obsid'])
     for obs in recent_obs:
-        process_obsids([obs['obsid']])
+        process_obsids([obs])
```

Comment:
I'm also not entirely clear on why this routine didn't originally pass the list of obsids to process_obsids and have this be a single function call.  But probably better to do the minimal change to make this run for now and improve it after I add regression testing.